src/assembler/encoder.ts

```diff
@@ -209,8 +209,7 @@ function isDataInstruction(instruction: IInstruction): boolean {
 function isSymbolDataInstruction(instruction: IInstruction): boolean {
   return (
     instruction.name == 'DCD' &&
```

Comment:
Isn't this comma separated behavior also possible with `DCW` and `DCB`?

Reply:
I don't think so. A symbol has always a value of 32 bits (word). So, it doesn't make much sense to only use `DCB` or `DCW`. But I haven't tried it out on the actual CT board yet.

Reply:
![grafik](https://user-images.githubusercontent.com/56474048/168412823-ebe4d260-8b00-4256-822f-36b6ca30022d.png)
does only work with DCD